spring-integration-file/src/main/java/org/springframework/integration/file/remote/synchronizer/AbstractInboundFileSynchronizer.java

```diff
@@ -233,6 +233,12 @@ public void synchronizeToLocalDirectory(final File localDirectory) {
 
 	@Override
 	public void synchronizeToLocalDirectory(final File localDirectory, final int maxFetchSize) {
+		if (maxFetchSize == 0) {
+			if (logger.isDebugEnabled()) {
+				logger.debug("Max Fetch Size is zero - fetch ignored");
```

Comment:
I'd say that it is WARN level. Even if `maxFetchSize == 0` is intentional, it might be better to warn about such an action.
WDYT?